Makefile

```diff
@@ -48,6 +49,12 @@ bin/atc: src/log/*.go src/atc/*.go
 	@mkdir -p bin
 	$(BUILD) -o $@ ${SRC}/atc
 
+jsx: src/react/jsx/*.js
+	@cd src/react && $(NPM) run build-js
```

Comment:
yeah, I dont have a way to run npm from the right directory (a la -C with tar for instance).
Each makefile line is ran in a sub-shell so we would not be in the right directory if we ran 1 command per line.
I will remove the @ though